tests/framework-tests/preferences/js/AuxBuilderTests.js

```diff
@@ -393,6 +392,70 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     });
 
     /*******************************************************************************
+<<<<<<< HEAD
```

Comment:
One feature that is completely undesirable in my 2007 JavaScript-editing Eclipse plugin is comment folding ....